### PR TITLE
[Modular] Ashwalker Village + Ashwalker Garb

### PR DESCRIPTION
**Pull request by ForrestWick**
## About The Pull Request

As Citadel did a long while ago, and as we had when Skyrat was still Citcode, Ashwalkers have been given an updated version of their nest in modular_skyrat. The actual nest ruin that was previously there is untouched, and if this winds up getting merged, I'll leave it up to whoever controls map stuff as to whether or not the small nest ruin gets left alone. If it does, I'll come back to it to add some of the QOL upgrades the village has. This PR also adds some new clothing options for Ashwalkers, supplied in the temple roundstart.

The village layout is seen here: https://cdn.discordapp.com/attachments/845865714979176551/864671876047700038/unknown.png

So far it's basic, very empty, kinda blocky. This is intentional, as the idea is to let Ashwalker players build the village up on their own, with only three set buildings (the tendril temple, the storage hut, and the cooking hut) as being the main, pre-built areas. The entire village is storm-proof and sheltered, because setting up areas in game every round is kinda annoying.

The temple now houses most of the tools the walkers need for QOL, as well as some replacements as well. The normal glass-tip spears are replaced by much more fitting bone-spears, with two tribal bows being given to the walkers roundstart, accompanied by two quivers and a handful of arrows. In addition, bone daggers are present in case those are more your character's style. Another toolbox has been added, as well as multiple belts because it's not fun for someone to run off with the only set of tools and die in God-knows-where. And to top it off, the MALF disk is in a safe again, because giving that power for free is kinda... Ehhhh...

Outside of the temple, we have the new grubber pen for the little crab things that eat gibs. These give milk like normal and are just in general kinda cute. There's also a dedicated farming area, and sandstone for you to make soil plots with. The storage hut has MANY more seeds for the aspiring lizard chefs wanting to make due with "limited" resources, as well as more botany tools, some starting resources, and some roundstart surgical tools.

In addition, an ORM has been added. See the cult forge outside the temple? It's an ORM with var-edited icon states. Now you don't have to rely on RNG to not fuck you over when you try smelting hard-earned ores. And the points can be used for barter with miners/Syndis! The hut down below from the Temple is also outfitted with some kitchen appliances. I couldn't figure out how to make these fit aesthetically, but I doubt anyone will care because non-powergame QOL that adds more to do in a role is always good. I also gave the ashwalkers some mining scanners, point cards, and a mining vendor to their storage hut so they can get some nice things with the mining points.

## Why It's Good For The Game

General QOL for Ashwalkers, a lot of it presents more options for how to do things. The map is modular and the normal tg ruin is untouched.. Also new clothes for the lizard dudes.

## Changelog
:cl:
add: Added more ashwalker clothes.
expansion: Made an Ashwalker village that is a permanent fixture of lavaland, expanded the previous ashwalker nest ruin as a base for this.
qol: Gives Ashwalkers more tools, barter options, and general gameplay options for their rounds.
/🆑
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/clothing/suits/misc.dm [hunk 1]
```diff
@@ -287,28 +287,19 @@
...
+
+/obj/item/clothing/suit/ashwalkermantle/cape
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
+	name = "brown leather cape"
+	desc = "An ash coated cloak."
+	icon_state = "desertcloak"
 	mutant_variants = NONE
 
-/obj/item/clothing/suit/corgisuit/en/New()
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/customization/modules/clothing/suits/misc.dm:297]: Unrelated changes here

Comment (ForrestWick) [reply modular_skyrat/modules/customization/modules/clothing/suits/misc.dm:297]: I have no idea what's going on there tbh

Comment (ForrestWick) [reply modular_skyrat/modules/customization/modules/clothing/suits/misc.dm:297]: More specifically not sure how to fix it. Amelia said it'd be fine

**Comment by ForrestWick** [comment]
Shit's fucked, gotta redo it again
